app/components/doi-subject.js

```diff
@@ -0,0 +1,139 @@
+import Component from '@ember/component';
+import ISO6391 from 'iso-639-1';
+import URI from 'urijs';
+import { isBlank } from '@ember/utils';
+// import OECD from 'helpers/oecd-list';
+// import { subjectList as OECD } from 'bracco/helpers/oecd-list';
+
+// const subjectList = OECD.subjects;
+const subjectList = [
+  'Natural sciences',
+  'Mathematics',
+  'Computer and information sciences',
+  'Physical sciences',
+  'Chemical sciences',
+  'Earth and related Environmental sciences',
+  'Biological sciences',
+  'Other natural sciences',
+  'Engineering and technology',
+  'Civil engineering',
+  'Electrical engineering, Electronic engineering, Information engineering',
+  'Mechanical engineering',
+  'Chemical engineering',
+  'Materials engineering',
+  'Medical engineering',
+  'Environmental engineering',
+  'Environmental biotechnology',
+  'Industrial biotechnology',
+  'Nano-technology',
+  'Other engineering and technologies',
+  'Medical and Health sciences',
+  'Basic medicine',
+  'Clinical medicine',
+  'Health sciences',
+  'Medical biotechnology',
+  'Other medical sciences',
+  'Agricultural sciences',
+  'Agriculture, Forestry, and Fisheries',
+  'Animal and Dairy science',
+  'Veterinary science',
+  'Agricultural biotechnology',
+  'Other agricultural sciences',
+  'Social sciences',
+  'Psychology',
+  'Economics and Business',
+  'Educational sciences',
+  'Sociology',
+  'Law',
+  'Political science',
+  'Social and economic geography',
+  'Media and communications',
+  'Other social sciences',
+  'Humanities',
+  'History and Archaeology',
+  'Languages and Literature',
+  'Philosophy, Ethics and Religion',
+  'Arts (arts, history of arts, performing arts, music)',
+  'Other humanities',
+];
+
+const oecdScheme = 'OECD';
+const oecdSchemeUri = 'http://www.oecd.org/science/inno/38235147.pdf';
+
+export default Component.extend({
+  subjectList,
+  subjects: subjectList,
```

Comment:
consider a better name for this as at first glance I wondered why we have two subjectList's
maybe:
completeSubjects
filteredSubjects